src/vm/peimagelayout.cpp

```diff
@@ -172,7 +183,9 @@ void PEImageLayout::ApplyBaseRelocations()
                 dwOldProtection = 0;
             }
 
-            IMAGE_SECTION_HEADER *pSection = RvaToSection(rva);
+            USHORT fixup = fixupsCount ? VAL16(fixups[0]) : 0;
```

Comment:
@jkotas The assertions on lines 162 and 163 effectively guarantee fixupsCount is non-zero.  I suspected it was not supposed to happen, but I was not sure, but now that I see the asserts, I know it is designed to not happen.  

Therefore I think it is OK to just simplify line 186.  Otherwise I can move line 242 to 159 and add an ```if (fixupsCount ==0) continue```, but it is unnecessary code complexity